.github/pull_request_template.md

```diff
@@ -43,70 +36,21 @@ Common items include:
 -->
 
 If you're reviewing this PR, please check for these things in particular:
-<!-- Add some items to the following list here -->
-* Verify all PR security questions and checklists have been completed and addressed.
-
+<!-- Add some items here -->
 
 ### What Security Implications Does This PR Have?
 
-Submitters should complete the following questionnaire:
-
-* If the answer to any of the questions below is **Yes**, then **you must** supply a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence here: **N/A**
+Please indicate if this PR does any of the following:  
 
-    * Does this PR add any new software dependencies? 
-      * [ ] Yes
-      * [ ] No
-    * Does this PR modify or invalidate any of our security controls?
-      * [ ] Yes
-      * [ ] No
-    * Does this PR store or transmit data that was not stored or transmitted before?
-      * [ ] Yes
-      * [ ] No
+* Adds any new software dependencies
+* Modifies any security controls
+* Adds new transmission or storage of data
+* Any other changes that could possibly affect secuirity? 
 
-* If the answer to any of the questions below is **Yes**, then please add @<!-- -->StewGoin as a reviewer, and note that this PR **should not be merged** unless/until he also approves it.
-    * Do you think this PR requires additional review of its security implications for other reasons?
-      * [ ] Yes
-      * [ ] No
+* [ ] Yes, one or more of the above security implications apply. This PR must not be merged without the ISSO or team security engineer's (@sb-benohe) approval. 
```

Comment:
I'd like to avoid check boxes that we don't expect to regularly be checked–github treats unchecked boxes as incomplete tasks in the PR interface. Could we do something like the following instead?
```suggestion
* [ ] I have considered the above security implications as it relates to this PR. (If one or more of the above apply, it cannot be merged without the ISSO or team security engineer's (@sb-benohe) approval.) 
```